lib/semirel.gi

```diff
@@ -720,13 +720,13 @@ function(semi)
 end);
 
 InstallOtherMethod(GreensHClasses, "for a Green's Class", true, [IsGreensDClass], 0,
```

Comment:
It might be better to `DeclareAttribute("GreensHClasses", IsGreensDClass);` and drop the `Other` here, but maybe there is some reason I don't know for doing it this way.

Also I believe the `true` and the `0` are redundant, and could be removed.